Makefile

```diff
@@ -160,3 +134,83 @@ golangci-lint-fix.%: $(GOLANGCI_LINT)
 
 .PHONY: FORCE # use this to force phony behavior for targets with pattern rules
 FORCE:
+
+##@ Deployment
+
+.PHONY: cluster
+cluster: kind ctlptl ## Create Kind cluster and local registry
+	$(CTLPTL) apply -f ctlptl.yaml
+
+.PHONY: cluster-reset
+cluster-reset: kind ctlptl ## Delete Kind cluster
+	$(CTLPTL) delete -f ctlptl.yaml
```

Comment:
These targets seem pretty simple. Can you remind me why we need to use ctlptl instead of using kind directly?

Reply:
The difference is, that KinD still does not have integrated registry (see https://github.com/kubernetes-sigs/kind/issues/1213) and requires [a workaround](https://kind.sigs.k8s.io/docs/user/local-registry/). Ctlptl automates the process of creating and connecting the registry to the KinD.